observer/lib/platform-stats.js

```diff
@@ -1,5 +1,5 @@
 /**
- * @param {import('pg').Client} pgClient
+ * @param {import('pg').ClientBase} pgClient
```

Comment:
for my understanding, why was this change necessary?

Reply:
I'll revert this change. Good catch!

I was experimenting with alternatives to the missing `Queryable` interface (see https://github.com/filecoin-station/spark-stats/pull/144#discussion_r1637976607) and forgot to revert this place.